### ISSUE TITLE
How to use this module to load the pre-trained model offered by the official PyTorch?

### ISSUE THREAD

**Issue by yu-changqian** [vacancy/Synchronized-BatchNorm-PyTorch#6]
If I want to use this module on the base model, how can I load the pre-trained model offered by the official PyTorch?
Or, should I train the base model on the ImageNet from scratch with this sync-bn?

**Comment by acgtyrant** [comment]
Just use `torch.load`, it use the string key to match module, and assign the tensor to the same size parameter, no matter what Module it is.

直接用 torch.load, 它只靠 str key 匹配，只要 Tensor 的大小和类型对得上，网络的参数就会直接被预训练模型的权重赋值，不管原来的 module 到底是什么 Module，又是否得为同一个 Module。load 后就可以直接训练了。

**Comment by acgtyrant** [comment]
You can reference this too: https://github.com/fyu/drn/blob/master/drn.py#L7

**Comment by yu-changqian** [comment]
Thank you. I get it.
